app/controllers/blogs_controller.rb

```diff
@@ -35,7 +35,6 @@ def edit
             @blog = @user.blogs.find_by(id:params[:id])
     end
     #@blog = Blog.find(params[:id])
```

Comment:
dont put parathesis in `if condition`
`if(session[:user_id])`  should be `if session[:user_id]`